app/src/main/res/xml/scene_event_details.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <Transition
+    app:constraintSetStart="@id/expanded"
```

Comment:
⚠️ this is dangerous naming, if we ever use another ML or want to call anything `collapsed` or `expanded` we're squatting the IDs. Maybe `event_details_expanded` and `event_details_collapsed` are better names